sources/Inputs/src/NetworkManager.cpp

```diff
@@ -163,8 +163,7 @@ NetworkManager::buildTree() {
     } else {
       converterType = HvdcLine::ConverterType::LCC;
     }
-    auto hvdcLineCreated =
-        boost::make_shared<dfl::inputs::HvdcLine>(hvdcLine->getID(), converterType, converter_1, converter_2);
+    auto hvdcLineCreated = boost::make_shared<dfl::inputs::HvdcLine>(hvdcLine->getID(), converterType, converter_1, converter_2);
```

Comment:
why use boost shared_ptr here ? it is better to use std::shared_ptr when we can

Reply:
I used boost shared_ptr because I though it was the norm in this project, is it not ?

Reply:
No, it is a requirement from dynawo because we don't have access to c++11 in dynawo so we can use only shared pointers from boost. But in DFL, we are in c++11

Reply:
I already use shared_ptr in other places in DFL code (see Algo)